iodata/iodata.py

```diff
@@ -259,22 +260,31 @@ def atcorenums(self) -> np.ndarray:
             # https://github.com/PyCQA/pylint/issues/1694
             # pylint: disable=no-member
             result = self.atnums.astype(float)
-            self._atcorenums = result
+            self.atcorenums = result
```

Comment:
(Comments are only possible on lines that have changed, which is a bit unfortunate, because sometimes new changes imply things about old code.)

Agreed, `result` can be eliminated, making things shorter. I'm not sure why it was this way, probably related to some old code.